awx/ui/client/lib/components/cards/_index.less

```diff
@@ -0,0 +1,52 @@
+.content-padding {
+    padding: 20px;
+}
+
+/* Panels */
+.twr-panel-title {
+    font-size: 1em;
+    color: #707070;
+    font-weight: bold;
+    line-height: 1em;
+}
+
+.twr-panel-text {
+    font-size: 0.9em;
+    color: #161b1f;
+}
+
+
+/* Card */
+
+.twr-card-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-gap: 20px;
+}
+
+.twr-card {
+    display: flex;
```

Comment:
My understanding of CSS-Grid is minimal at best, but if the parent container, in this case `twr-card-container` uses `display:grid`, then does it's child elements need to specify `display:flex`? I thought that the parent element would take care of all that for you.

Reply:
Seems kind of redundant to have css-grid and flex box in motion here. Maybe the flex is for the content in the card itself?